src/kernel-install/kernel-install.c

```diff
@@ -1127,9 +1127,13 @@ static int help(void) {
         printf("%1$s [OPTIONS...] COMMAND ...\n\n"
                "%2$sAdd and remove kernel and initrd images to and from /boot%3$s\n"
                "\nUsage:\n"
-               "  %1$s [OPTIONS...] add KERNEL-VERSION KERNEL-IMAGE [INITRD-FILE...]\n"
-               "  %1$s [OPTIONS...] remove KERNEL-VERSION\n"
-               "  %1$s [OPTIONS...] inspect [KERNEL-IMAGE]\n"
+               "  kernel-install [OPTIONS...] add KERNEL-VERSION KERNEL-IMAGE [INITRD-FILE...]\n"
+               "  kernel-install [OPTIONS...] remove KERNEL-VERSION\n"
+               "  kernel-install [OPTIONS...] inspect [KERNEL-IMAGE]\n"
+               "\n"
+               "This program may also be invoked as 'installkernel':\n"
```

Comment:
(here too, I'd move this to the bottom of the --help text, 

And maybe say "For compatibility with the Linux kernel build system"